mdata/aggmetric_test.go

```diff
@@ -37,13 +45,24 @@ func (c *Checker) Add(ts uint32, val float64) {
 	c.points = append(c.points, point{ts, val})
 }
 
+func (c *Checker) DropPointByTs(ts uint32) {
+	for i := 0; i != len(c.points); {
+		if c.points[i].ts == ts {
+			c.points = append(c.points[:i], c.points[i+1:]...)
+		} else {
+			i++
+		}
+	}
+}
+
 // from to is the range that gets requested from AggMetric
 // first/last is what we use as data range to compare to (both inclusive)
 // these may be different because AggMetric returns broader rangers (due to packed format),
 func (c *Checker) Verify(primary bool, from, to, first, last uint32) {
 	currentClusterStatus := cluster.Manager.IsPrimary()
+	sort.Sort(ByTs(c.points))
```

Comment:
Checker.Add has a comment saying to add points in asc order, which doesn't seem correct anymore.